lang/elixir/Portfile

```
@@ -39,3 +39,7 @@ destroot.args       PREFIX=${prefix}
 
 # various tests have been removed using above patch
 test.run            yes
+
+livecheck.type  regex
+livecheck.url   ${github.homepage}/releases
+livecheck.regex v(\[0-9\]+\\.\[0-9\]+\\.\[0-9\]+)
```

```suggestion
livecheck.regex v(\\d+\\.\\d+\\.\\d+)
```

This should be equivalent and be more readable, wdyt?

---

Added @herbygillot suggestion after testing it.